packages/rum/src/domain/segmentCollection/segmentCollection.spec.ts

```diff
@@ -123,28 +118,6 @@ describe('startSegmentCollection', () => {
       expect(sendCurrentSegment().creation_reason).toBe('view_change')
     })
 
-    it('flushes segment when the page become hidden', () => {
-      setPageVisibility('hidden')
-      const { addRecord, eventEmitter, sendCurrentSegment, worker, sendSpy } = startSegmentCollection(CONTEXT)
-      addRecord(RECORD)
-      eventEmitter.dispatchEvent(createNewEvent(DOM_EVENT.VISIBILITY_CHANGE))
-
-      worker.processAllMessages()
-      expect(sendSpy).toHaveBeenCalledTimes(1)
-      expect(sendCurrentSegment().creation_reason).toBe('visibility_hidden')
-    })
-
-    it('does not flush segment when the page become visible', () => {
-      setPageVisibility('visible')
-      const { addRecord, eventEmitter, sendCurrentSegment, worker, sendSpy } = startSegmentCollection(CONTEXT)
-      addRecord(RECORD)
-      eventEmitter.dispatchEvent(createNewEvent(DOM_EVENT.VISIBILITY_CHANGE))
-
-      worker.processAllMessages()
-      expect(sendSpy).not.toHaveBeenCalled()
-      expect(sendCurrentSegment().creation_reason).not.toBe('visibility_hidden')
-    })
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Should not we keep those cases? with `{ isUnloading: false }`